assets/js/modules/analytics/datastore/accounts.test.js

```diff
@@ -748,62 +841,60 @@ describe( 'modules/analytics accounts', () => {
 					registry.select( MODULES_ANALYTICS ).getProfileID()
 				).toBe( matchedProperty.defaultProfileId ); // eslint-disable-line sitekit/acronym-case
 			} );
-
-			describe( 'analytics-4', () => {
-				beforeEach( () => {
-					enabledFeatures.add( 'ga4setup' );
-
+		} );
+		describe( 'getAccounts - analytics-4', () => {
```

Comment:
I had to break this describe block out of the previous as all the extra `fetchMocks` I made were interfering with these ones.